### ISSUE TITLE
IPv6 Cluster - Test 2.10.5.rc2

### ISSUE THREAD

**Issue by feature86** [telepresenceio/telepresence#3005]
Hi!

I was testing the 2.10.5.rc2 in an IPv6 only cluster to verify the changes made in the Ticket [2978](https://github.com/telepresenceio/telepresence/issues/2978 )
I still have issues with the IPv6 eks cluster. 

-  No routes to the pod network (Still invalid argument error) 
-  No route to the service Subnet  (see the log file, I think the `t2-tst-dummy` is IPv4 only. 

```2023-02-03 12:01:18.5921 info    Traffic Manager v2.10.5-rc.2 [uid:1000,gid:0]
2023-02-03 12:01:18.5933 info    unable to load license: error reading license: open /home/telepresence/license: no such file or directory
2023-02-03 12:01:18.5934 info    starting cloud token watchers
2023-02-03 12:01:18.6164 info    Using cluster domain "cluster.local."
2023-02-03 12:01:18.6207 error   unable to extract service subnet from error message "Service \"t2-tst-dummy\" is invalid: spec.clusterIPs[0]: Invalid value: []string{\"1.1.1.1\"}: IPv4 is not configured on this cluster"
2023-02-03 12:01:18.6207 warning Service "t2-tst-dummy" is invalid: spec.clusterIPs[0]: Invalid value: []string{"1.1.1.1"}: IPv4 is not configured on this cluster
2023-02-03 12:01:18.6267 warning traffic manager is not able to resolve the IP of its own service
2023-02-03 12:01:18.6269 info    Using podCIDRStrategy: auto 
```

I hope this will help to improve IPv6 cluster support. 

Kind Regards!

**Comment by olgkpln** [comment]
Is there a planned fix on this matter? we are suffering from the exact same problem.

**Comment by Strangerxxx** [comment]
same issue on ipv6-only EKS completely blocks of using telepresence :(

@feature86 @olgkpln have you guys managed to overcome it somehow?

**Comment by thallgren** [comment]
Closed by #3486
